packages/puppeteer-core/src/api/locators/locators.ts

```diff
@@ -113,6 +114,18 @@ export abstract class Locator<T> extends EventEmitter<LocatorEvents> {
     return RaceLocator.create(locators);
   }
 
+  /**
+   * Trying to locate all the elements for locator array in parallel
+   * and return the last locator that locate element.
+   *
```

Comment:
could you please add an inline `@example` on how to use the locator to this comment?